syntax = "proto3";
package toolkit;




import "cmdb_extend_sdk/model/cmdb_extend/toolkit.proto";

/*
ListToolkit请求
*/
message ListToolkitRequest {
    /*
    模型Id
    */
    string objectId = 1 ;
}

/*
ListToolkit返回
*/
message ListToolkitResponse {
    /*
    模型Id
    */
    string objectId = 1 ;
    /*
    状态
    */
    bool enabled = 2 ;
    /*
    任意门列表
    */
    repeated cmdb_extend.Toolkit toolList = 3 ;
}

/*
ListToolkitApi返回
*/
message ListToolkitResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    toolkit.ListToolkitResponse data = 4 ;
}

